deployment-template/ecs/aws-otel-ec2-instance-metrics-daemon-deployment-cfn.yaml

```diff
@@ -60,6 +60,7 @@ Resources:
       ContainerDefinitions:
         - Name: aws-collector
           Image: 'public.ecr.aws/aws-observability/aws-otel-collector:latest'
+          Privileged: true
```

Comment:
Use `user` field instead of priveleged to set to the root user.

Reply:
I have reverted the changes made to this deployment template as it didn't require root user access. This template is only for instance metrics and not for container/task level metrics and hence don't require root access.